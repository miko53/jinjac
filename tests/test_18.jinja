{% for user in users %}-
This is user {{ user }}
{% endfor %}
dfdsg
