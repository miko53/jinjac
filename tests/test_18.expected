-
This is user dana
-
This is user bob
-
This is user john

dfdsg
